Makefile.work

```diff
@@ -176,6 +177,7 @@ SONIC_BUILD_INSTRUCTION :=  make \
                            ENABLE_DHCP_GRAPH_SERVICE=$(ENABLE_DHCP_GRAPH_SERVICE) \
                            ENABLE_ZTP=$(ENABLE_ZTP) \
                            SHUTDOWN_BGP_ON_START=$(SHUTDOWN_BGP_ON_START) \
+                           INSTALL_KUBERNETES=$(INSTALL_KUBERNETES) \
```

Comment:
>INSTALL_KUBERNETES=$(INSTALL_KUBERNETES) [](start = 27, length = 40)

These config are supporting either from command line environment variable or rule.confg. Did you support?

Reply:
Yes from both. Let me test to confirm.

Reply:
Tested to work,